.travis.yml

```diff
@@ -17,13 +17,19 @@ cache:
     directories:
         - $HOME/.composer/cache
 
+before_script:
+    - echo 'phar.readonly = "0"' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
 before_install:
     - sudo add-apt-repository -y ppa:git-core/ppa
     - sudo apt-get update && sudo apt-get install git-man && sudo apt-get install -y git
     - git --version
     - phpenv config-rm xdebug.ini || true
+    - export PATH="$PATH:$HOME/.composer/vendor/bin"
 
-install: composer update --prefer-dist --no-interaction --optimize-autoloader
+install:
+    - composer global require kherge/box --prefer-dist --no-interaction --optimize-autoloader
```

Comment:
@ogizanagi I given it a try when nervo introduced the phar. The problem comes from https://packagist.org/packages/kherge/amend#3.0.4 which is required by `kherge/box` and is locked to `symfony/console: ~2.1`. However the master branch is ok, I will open an issue for requesting a new tag.